### ISSUE TITLE
[Starter Projects] Issue Name

### ISSUE THREAD

**Issue by AmaZili** [zurb/foundation-sites-6#241]
**How can we reproduce this bug?**

1. start a powershell on windows 10
2. PS D:\> mkdir project
3. PS D:\> cd project
4 PS D:\project> foundation new --framework sites --template advanced

**What did you expect to happen?**
Have foundation 6 installed in the directory
**What happened instead?**
ERROR: "foundation new" was called with arguments ["--framework", "sites", "--template", "advanced"]
Usage: "foundation new"
PS D:\project> foundation new
ERROR: "foundation new" was called with no arguments
Usage: "foundation new"
**Test case**
PS D:\project> node -v
v5.0.0
PS D:\project> npm version
{ npm: '3.3.6',
  ares: '1.10.1-DEV',
  http_parser: '2.5.0',
  icu: '56.1',
  modules: '47',
  node: '5.0.0',
  openssl: '1.0.2d',
  uv: '1.7.5',
  v8: '4.6.85.28',
  zlib: '1.2.8' }

**Comment by zurbchris** [comment]
It's looking for a project name
```bash
foundation new [project name] --framework ...
```

**Comment by AmaZili** [comment]
Unfortunatly : 

PS D:\> foundation new test --famework sites --template advanced
ERROR: "foundation new" was called with arguments ["test", "--famework", "sites", "--template", "advanced"]
Usage: "foundation new"
PS D:\>

**Comment by joepurdy** [comment]
I'm going to take a guess and say you're using the ruby gem foundation CLI tool. To confirm try running `where foundation` if you have the ruby gem you'll see something like this:

```
C:\Users\<username>
λ where foundation
c:\Ruby22-x64\bin\foundation
c:\Ruby22-x64\bin\foundation.bat
```

The new foundation 6 cli can be installed with npm using the following command:
`npm install -g foundation-cli@2.0.0-beta.2`

Based on what is reported in issue #228 you need to be using the 0.12 branch of node with the v2.x branch of npm.

Assuming you get switched over from the gem to the node cli you should be able to run the new command, you may also have to uninstall the foundation ruby gem with `gem uninstall foundation` before installing the npm foundation-cli.

**Comment by gakimball** [comment]
Yep, your issue is that you have the old `foundation` command on your system. There's no easy way around that, because we want people to switch to the new one eventually (although it's not going to have any F5 support).

It's relatively easy to `gem uninstall foundation` and `gem install foundation` later, but if you use the F5 CLI a lot then I understand that's going to be a hassle.

**Comment by gakimball** [comment]
Temporarily uninstalling one CLI or the other is going to resolve your issue. With the way your machine is configured, commands installed by Ruby have precedence over commands installed by Node, which is why the `foundation` command still points to the Foundation 5 CLI for you.

We don't have plans to have the two command line tools sit side-by-side.

**Comment by AmaZili** [comment]
Sorry not to have seen multiple replies before (out of town).
Yes We use the gen/ruby for foundation rev 3 to 5, started to work on rev 6 and fell in this.

How do you propose to work with compatibility ? Most agencies will have projects in different revisions just like us ...

**Comment by gakimball** [comment]
@AmaZili You don't need the CLI to run existing projects, just to make new ones. However, if you see yourself needing to make new 5 and 6 projects in the near future, it's possible to create a command line alias to one of the commands.

It looks like you're a Windows user, and I haven't used the Windows shell in a while. What command line environment do you use?

**Comment by AmaZili** [comment]
@gakimball (windows, npm, node + Ruby) We are running both old compass and more recently grunt cli, moving to gulp with rev 6

**Comment by gakimball** [comment]
@AmaZili Do you use Command Prompt, or some other Windows shell like cmder?

**Comment by AmaZili** [comment]
We use windows prompt or sometimes powershell

**Comment by AmaZili** [comment]
For the tests, did : 
gem uninstall foundation
npm install -g foundation-cli@2.0.0-beta.2
foundation new

And had it to work.

Now need to figure how to manage with maintaning older projects...

**Comment by gakimball** [comment]
@AmaZili We've published a finished version of the CLI, just `2.0.0`. See if that one works any better for you.
